Project.toml

```diff
@@ -8,7 +8,6 @@ FileIO = "5789e2e9-d7fb-5bc7-8068-2c6fae9b9549"
 Tables = "bd369af6-aec1-5ad0-b16a-f7cc5008161c"
 
 [compat]
-FileIO = "1.4"
 Tables = "1.2, 1.1"
```

Comment:
Given that this package is not yet registered, let's fix the latest stable release of Tables to 1.2. Please remove the 1.1 entry.